airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -848,10 +840,6 @@ public void testIncrementalDedupeSync() throws Exception {
    */
   @Test
   void testSyncVeryBigRecords() throws Exception {
```

Comment:
I think you need to delete this method entirely - previously all destinations except redshift were skipping it (and now we also want redshift to skip it)